channels/stable-4.5.yaml

```diff
@@ -1,6 +1,20 @@
 name: stable-4.5
 versions:
 
+- 4.4.16
```

Comment:
lets drop this for now since #372 hasn't merged yet

Reply:
Fixed it.